Add hidden context to Search nav

PBruk opened the pull request:
Tagging @frrrances for code-review: resolves #67.

frrrances commented on style.css:
```diff
@@ -498,10 +498,12 @@ img.wp-smiley,
 /* Assistive text */
 .assistive-text,
-.site .screen-reader-text {
+.site .screen-reader-text,
```
why the double class? and can we change this class to ".sr" for screenreader?

frrrances commented on style.css:
```diff
@@ -497,11 +497,11 @@ img.wp-smiley,
 /* Assistive text */
-.assistive-text,
-.site .screen-reader-text {
+.assistive-text {
```
Did you mean for this class to not match the html?

PBruk commented:
IIRC, line 501 was created when I added .screen-reader-text class. The reason I prefixed .site was because the styling didn't work, otherwise. I removed it now, since I am removing that class from the templates I added it to but am leaving .assistive-text because I don't if that still exists somewhere, so I'd rather not break anything.

frrrances commented:
gotcha - makes sense! thanks for the extra info.

frrrances commented:
👍
